controllers/managedocs_controller.go

```diff
@@ -48,28 +47,29 @@ import (
 	"github.com/openshift/ocs-osd-deployer/templates"
 	"github.com/openshift/ocs-osd-deployer/utils"
 	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	promv1a1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1alpha1"
```

Comment:
Its required to register `promv1a1` in the scheme

`utilruntime.Must(promv1a1.AddToScheme(scheme))` (main.go)